mocha.js

```diff
@@ -6186,30 +6186,30 @@ var clearInterval = global.clearInterval;
  * to allow running node code in
  * the browser.
  */
+var process = (global && global.process) || {};
```

Comment:
Add:

// practicalmeteor change - use real node api process in server side code